### ISSUE TITLE
[Accessibility]: Copy clipboard button overlaps with repo dropdown and path is not visible

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36629]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34448

### Problem description

See the attached screenshot. The path for the file with links is not visible, but the copy clipboard button which is usually at the trailing end of the path is somehow visually overlapping with the repo dropdown.

### Expected behavior

If the copy to clipboard button is visible, it should have space around it so that it can be confidently clicked. Similarly for the file path links.

IDK if all of these can visually fit though; maybe they should just be hidden? 🤔

### Additional details

![screenshot of 320px wide screen](https://user-images.githubusercontent.com/93103176/172118053-f0c78e8f-77be-47f5-afa4-a21ba892e27f.png)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
